--- infos_sondage.php.ORI.php	Thu Apr 29 15:19:20 2010
+++ infos_sondage.php	Thu Apr 29 17:07:36 2010
@@ -37,6 +37,9 @@
 //
 //==========================================================================
 
+#AJOUT
+$FQDN="@lyceenobel.org";
+
 session_start();
 include 'fonctions.php';
 if (file_exists('bandeaux_local.php'))
@@ -204,7 +207,7 @@
 echo '</tr>'."\n";
 echo '<tr><td>'.$tt_infos_champ_nom.'</td><td>';
 if (isset($_SERVER['REMOTE_USER']))
-	echo '<input type="hidden" name="nom" size="40" maxlength="40" value="'.$_SESSION["nom"].'">'.$_SESSION["nom"].'</td>'."\n";
+	echo '<input type="hidden" name="nom" size="40" maxlength="40" value="'.$_SERVER['REMOTE_USER'].'">'.$_SESSION["nom"].'</td>'."\n";
 else
 	echo '<input type="text" name="nom" size="40" maxlength="40" value="'.$_SESSION["nom"].'"></td>'."\n";
 if (!$_SESSION["nom"]&&($_POST["creation_sondage_date"]||$_POST["creation_sondage_autre"]||$_POST["creation_sondage_date_x"]||$_POST["creation_sondage_autre_x"])){
@@ -216,7 +219,7 @@
 echo '</tr>'."\n";
 echo '<tr><td>'.$tt_infos_champ_adressemail.'</td><td>';
 if (isset($_SERVER['REMOTE_USER']))
-	echo '<input type="hidden" name="adresse" size="40" maxlength="64" value="'.$_SESSION["adresse"].'">'.$_SESSION["adresse"].'</td>'."\n";
+	echo '<input type="hidden" name="adresse" size="40" maxlength="64" value="'.$_SERVER['REMOTE_USER'].$FQDN.'">'.$_SESSION["adresse"].'</td>'."\n";
 else
 	echo '<input type="text" name="adresse" size="40" maxlength="64" value="'.$_SESSION["adresse"].'"></td>'."\n";
 if (!$_SESSION["adresse"]&&($_POST["creation_sondage_date"]||$_POST["creation_sondage_autre"]||$_POST["creation_sondage_date_x"]||$_POST["creation_sondage_autre_x"])){
